src/value_generators/src/DocumentGenerator.cpp

```diff
@@ -65,12 +63,16 @@ class Generator : public Appendable {
 
 namespace {
 using namespace genny;
-const static boost::posix_time::ptime epoch{boost::gregorian::date(1970, 1, 1)};
+
```

Comment:
I feel that I have somewhat contributed to some of the formatting issues here. Will evergreen.yml give errors for these issues in future?